### PR TITLE
Release 0.1.x

### PR DESCRIPTION
**Pull request by alipatton10**
Added line kubectl create namespace dynatrace to setupInfrastructure.sh as required by the new v0.3.0 oneagent operator as documented https://www.dynatrace.com/support/help/technology-support/cloud-platforms/kubernetes/installation-and-operation/full-stack/deploy-oneagent-on-kubernetes/
Granted execute permissions to applyRequestAttributeRules.sh
Added temp --validate=false to the kubectl create -f https://raw.githubusercontent.com/Dynatrace/dynatrace-oneagent-operator/$LATEST_RELEASE/deploy/kubernetes.yaml as there is an issue with the GKE validation. This should be removed again once it is fixed

### CODE REVIEW

**Comment by johannes-b** [comment]
Thanks for adding 'create namespace'.

In order to merge the pull request, we identified a security restriction from Travis-CI. A Travis-CI build does not allow to access secure environment variables when a PR comes from a fork. (In this particular case, the PR comes from: **dt-acm-keptn**:release-0.1.x)

To come around the security limitation, I had to merge the changes from dt-acm-keptn:release-0.1.x into an internal branch: keptn:release-0-1-merge. Afterwards, I created a PR from this internal branch to keptn:release-0.1.x that can be build using Travis-CI.

Long story short, changes are now in keptn:release-0.1.x and we are looking for a solution to deal with external PRs.

**Comment by alipatton10** [comment]
Could you not just add me to the repository?
